### PR TITLE
Support Unicode Big Operators

### PR DESCRIPTION
**Pull request by ronkok**
This is the sixth in a series of PRs to give KaTeX the ability to recognize Unicode character input.

This is the final PR in this series that will submit characters in bulk. After this, there come only a few characters whose alias or atom type was not obvious. I'll turn in PRs for some of them, too, but only after they are vetted.

### CODE REVIEW

**Comment by ronkok** [comment]
As in PR #933, the following table shows: (1) the KaTeX function to which each character is mapped, (2) confirmation whether [unicode math](http://ctan.sharelatex.com/tex-archive/macros/latex/contrib/unicode-math/unicode-math-table.tex) and/or John Cook’s [conversion page](https://www.johndcook.com/unidata.js) map it the same way, (3) possible alternates that I did not use.

  

| **Character** | **Code point** | **Maps to** | **unicode math** | **Cook** | **Not used**        |
|:-------------:|:--------------:|:------------|:----------------:|:--------:|:-------------------:|
| ∏             | U+220F         | \\prod      | ✓                | ✓        |                     |
| ∐             | U+2210         | \\coprod    | ✓                | ✓        |                     |
| ∑             | U+2211         | \\sum       | ✓                | ✓        |                     |
| ∫             | U+222B         | \\int       | ✓                | ✓        | \\intop, \\smallint |
| ∬             | U+222C         | \\iint      | ✓                | ✓        |                     |
| ∭             | U+222D         | \\iiint     | ✓                | ✓        |                     |
| ∮             | U+222E         | \\oint      | ✓                | ✓        |                     |
| ⋀             | U+22C0         | \\bigwedge  | ✓                | ✓        |                     |
| ⋁             | U+22C1         | \\bigvee    | ✓                | ✓        |                     |
| ⋂             | U+22C2         | \\bigcap    | ✓                | ✓        |                     |
| ⋃             | U+22C3         | \\bigcup    | ✓                | ✓        |                     |
| ⨀             | U+2A00         | \\bigodot   | ✓                | ✓        |                     |
| ⨁             | U+2A01         | \\bigoplus  | ✓                | ✓        |                     |
| ⨂             | U+2A02         | \\bigotimes | ✓                | ✓        |                     |
| ⨄             | U+2A04         | \\biguplus  | ✓                | ✓        |                     |
| ⨆             | U+2A06         | \\bigsqcup  | ✓                | ✓        |                     |

File: src/functions.js [hunk 1]
```diff
@@ -297,15 +297,24 @@ defineFunction([
...
 }, function(context) {
+    let fName = context.funcName;
+    if (fName.length === 1) {
+        fName = {
+            "∫": "\\int",
+            "\u222c": "\\iint",
+            "\u222d": "\\iiint",
+            "\u222e": "\\oint",
+        }[fName];
+    }
```
Comment (edemaine) [inline src/functions.js]: This feels a little weird, to have the constant in the code there.  It's a little inefficient if this gets run a lot (though perhaps slightly faster as is if it never gets used...).  Thoughts on moving it to a `const` outside the function?

File: src/functions/op.js [hunk 2]
```diff
@@ -221,17 +221,36 @@ defineFunction({
...
+                "\u22c1": "\\bigvee",
+                "\u22c2": "\\bigcap",
+                "\u22c3": "\\bigcap",
+                "\u2a00": "\\bigodot",
+                "\u2a01": "\\bigoplus",
+                "\u2a02": "\\bigotimes",
+                "\u2a04": "\\biguplus",
+                "\u2a06": "\\bigsqcup",
+            }[fName];
+        }
```
Comment (edemaine) [inline src/functions/op.js]: Same comment as above.

File: src/functions.js [hunk 3]
```diff
@@ -297,15 +297,24 @@ defineFunction([
 
 // No limits, symbols
 defineFunction([
-    "\\int", "\\iint", "\\iiint", "\\oint",
+    "\\int", "\\iint", "\\iiint", "\\oint", "∫", "\u222c", "\u222d", "\u222e",
```
Comment (edemaine) [inline src/functions.js]: Is there a reason that `∫` gets embedded in Unicode and the others are `\u` escape sequences?  Something about the character being available in a default font?

Comment (ronkok) [reply src/functions.js]: Your guess is on the money. I tend to code in the Consolas font, in which ∫ is available, but not the other characters in this PR. If you would like to see some of any of them changed, let me know.

File: src/functions.js (hunk 1 above)
Comment (ronkok) [reply src/functions.js]: Sure. I'll move it to a constant outside the function.

File: src/functions/op.js (hunk 2 above)
Comment (ronkok) [reply src/functions/op.js]: Ditto.

File: src/functions.js (hunk 3 above)
Comment (edemaine) [reply src/functions.js]: I feel like it'd be more consistent to put them all as `\u` escape sequences, but I don't feel strongly about it.

Comment (ronkok) [reply src/functions.js]: Done.

**Comment by edemaine** [comment]
Thanks! Maybe the wrong code, though because the test fails now.

**Comment by ronkok** [comment]
You would think that I could type ten keystrokes without a typo. But apparently not.

Fixed now.

**Comment by edemaine** [comment]
Yay! Unicode mission nearing completion!

**Comment by kevinbarabash** [comment]
@edemaine thanks for reviewing and merging.

**Pull request merged**